js/src/rive.ts

```diff
@@ -0,0 +1,965 @@
+import Runtime from '../../wasm/publish/rive.js';
```

Comment:
I think rive-canvas should be a dependency of rive-js in package.json as they are two different libs on npm. This would avoid including `rive-canvas` in the bundle size of `rive-js` and let app developer favorite builder do the magic. 
But I don't know how it would work if your use unpkg directly...

Also, you can use [webpack path plugin](https://www.npmjs.com/package/tsconfig-paths-webpack-plugin) to manage dependency between your projects more easily.